### ISSUE TITLE
Add a page explaining creating an account

### ISSUE THREAD

**Issue by hamidra** [hamidra/dotdrop#8]
We need to add a page explaining about polkadot accounts what they are and how they work. The target would be ordinary people who are creating an account for the first time and not very token savvy, Maybe explain what are polkadot accounts and how they can store them or a set of FAQs or both!

https://www.mycrypto.com/
https://www.myetherwallet.com/
https://wiki.polkadot.network/docs/en/learn-account-generation#create-account

**Comment by erler** [comment]
Something like this is very much needed. I'm just wondering if this should live in the same About / FAQ page.

There's also this:
https://support.polkadot.network/support/solutions/articles/65000098878-how-to-create-a-dot-account

and this:
https://wiki.polkadot.network/docs/en/learn-account-generation

So maybe in order to avoid too much duplicated documentation, our explanation is a brief overview and then we link to further documentation on the Polkadot Wiki.

**Comment by hamidra** [comment]
Right. It can be a section in the same about page that we can link to or a separate page. Agree, we should keep in light so the user can get a high level information about polkadot accounts and then for more info they can refer to wiki. There is also ideas  #44 about adding some info about the utility of DOT tokens to educate user about the stuff they can do with their DOTs. those can also be in the account page or we can put direct links in the account overview page to the wiki/info based on the utility (e.x. staking, voting, ....)
